ivy/functional/frontends/tensorflow/nn.py

```diff
@@ -482,3 +482,11 @@ def relu6(features, name=None):
 @to_ivy_arrays_and_back
 def softmax(logits, axis=None, name=None):
     return ivy.softmax(logits, axis=axis)
+
```

Comment:
Hi! A small lint error `E302 expected 2 blank lines, found 1` Could you please fix that really quick?

Reply:
sure